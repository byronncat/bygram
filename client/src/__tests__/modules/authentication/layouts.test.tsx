import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

import { AuthenticationLayout } from '@authentication';

describe('authentication layout', () => {
  it('should render correctly when not authenticated', () => {
    const { container } = render(<AuthenticationLayout title="login" />);
    expect(container).toMatchSnapshot();
  });
});
