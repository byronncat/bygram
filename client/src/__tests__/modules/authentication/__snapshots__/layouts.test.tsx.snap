// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthenticationLayout should render correctly when not authenticated 1`] = `
<div
  className="w-100 h-100 bg-slate-700 overflow-hidden user-select-none position-relative"
>
  <span
    className="w-100 h-100 d-block position-absolute z-n1"
    style={
      Object {
        "backgroundImage": "url(night-neon.avif)",
        "backgroundPosition": "center center",
        "backgroundSize": "cover",
      }
    }
  />
  <div
    className="form-side text-white position-relative float-end h-100 d-flex flex-column justify-content-center align-items-center"
  >
    <img
      alt="logo"
      className="d-block"
      height={60}
      src="logo.svg"
      width={60}
    />
    <h1
      className="brand-name text-neon-glowing-1 text-uppercase fs-2 mt-2 my-3"
    >
      bygram
    </h1>
    <span
      className="panel-bg w-100 h-100 position-absolute z-n1"
    />
    <span
      className="title text-neon-glowing-2 m-2 text-capitalize position-absolute start-0 top-0"
    >
      login
    </span>
  </div>
</div>
`;
